build.sh

```diff
@@ -410,15 +411,18 @@ function runWattsi {
 
 runWattsi "$HTML_TEMP/source-whatwg-complete" "$HTML_TEMP/wattsi-output"
 if [[ "$WATTSI_RESULT" == "0" ]]; then
+  if [[ "$LOCAL_WATTSI" != true ]]; then
     "$QUIET" || grep -v '^$' "$HTML_TEMP/wattsi-output.txt" # trim blank lines
+  fi
 else
-  grep -v '^$' "$HTML_TEMP/wattsi-output.txt" # trim blank lines
   if [[ "$WATTSI_RESULT" == "65" ]]; then
```

Comment:
Collapse these two into `elif`?

Reply:
```
 else
-  grep -v '^$' "$HTML_TEMP/wattsi-output.txt" # trim blank lines
   if [[ "$WATTSI_RESULT" == "65" ]]; then
```

In hindsight, I think it was a mistake to drop that `grep` line in this change — because it makes the remote-wattsi message-output behavior different from the local-wattsi behavior. So I pushed a changed that restores it).